db/migrate/20240111163859_update_member_access_levels.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# migration to update the access levels for members as a new role `Uploader` was added
+class UpdateMemberAccessLevels < ActiveRecord::Migration[7.1]
+  def change
+    members = Member.where(access_level: [Member::AccessLevel::ANALYST - 10,
```

Comment:
**nitpick (blocking):** Can you just define the old access levels as an enum here so that this always works even if we change the roles in the future.

Reply:
I defined the old access levels as a hash variable as the migration wouldn't let me use `enum`. I also tried to use `create_enum` but it kept complaining about wrong number of arguments even though 2 arguments were supplied.

Updated in d688b546aed2cabcc936f3c45af1fceca77083b3